rca/research/models.py

```diff
@@ -1,6 +1,313 @@
+                page = value.page.specific
+                research_spaces.append(
+                    {
+                        "title": page.title,
+                        "link": page.url,
+                        "image": page.listing_image,
+                        "description": page.introduction,
+                    }
+                )
+        return research_spaces
+
+    def get_research_opportunities(self):
+        research_opportunities = []
+        for value in self.research_opportunities.select_related("page"):
+            if value.page.live:
+                page = value.page.specific
+                research_opportunities.append(
+                    {
+                        "title": page.title,
+                        "link": page.url,
+                        "image": page.listing_image,
+                        "description": page.listing_summary,
+                    }
+                )
+        return research_opportunities
+
+    def get_research_news(self):
+        research_news = {"title": self.more_research_centre_content_title, "slides": []}
+        for value in self.research_news.select_related("page"):
+            if value.page.live:
+                page = value.page.specific
+                research_news["slides"].append(
+                    {
+                        "value": {
+                            "title": page.title,
+                            "link": page.url,
+                            "image": page.listing_image,
+                            "summary": page.listing_summary,
+                        }
+                    }
+                )
+        return research_news
+
+    def clean(self):
+        errors = defaultdict(list)
+
+        if self.twitter_username.startswith("@"):
+            self.twitter_username = self.twitter_username[1:]
+
+        if self.about_page and self.about_page_url:
+            errors["about_page"].append(
+                "Please choose between an internal page, or an external link"
+            )
+        if (
+            self.about_page
+            and not self.about_page_link_text
+            or self.about_page_url
+            and not self.about_page_link_text
+        ):
+            errors["about_page"].append("Please add some link text for the about page")
+        if self.staff_link and not self.staff_link_text:
+            errors["staff_link_text"].append(
+                "Please add some text for the link to all staff"
+            )
+
+        if errors:
+            raise ValidationError(errors)
+
+    def get_context(self, request, *args, **kwargs):
+        context = super().get_context(request, *args, **kwargs)
+        context["hero_colour"] = "dark"
+        if int(self.hero_colour_option) == LIGHT_TEXT_ON_DARK_IMAGE:
+            context["hero_colour"] = "light"
+
+        context["about_page"] = self.about_page_url
+        if self.about_page:
+            context["about_page"] = self.about_page.url
```

Comment:
These three lines could be:

`context["about_page"] = self.about_page.url if self.about_page else self.about_page_url`